packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -447,8 +448,8 @@ class Shortcuts extends StatefulWidget {
 }
 
 class _ShortcutsState extends State<Shortcuts> {
-  ShortcutManager _internalManager;
-  ShortcutManager get manager => widget.manager ?? _internalManager;
+  ShortcutManager? _internalManager;
```

Comment:
I think this can be `late`, and then the getter below can return a non-nullable value.

Reply:
It cannot be late because it can be set to null:
```
  @override
  void didUpdateWidget(Shortcuts oldWidget) {
    super.didUpdateWidget(oldWidget);
    if (widget.manager != oldWidget.manager) {
      if (widget.manager != null) {
        _internalManager?.dispose();
        _internalManager = null;
      } else {
        _internalManager ??= ShortcutManager();
      }
    }
    manager!.shortcuts = widget.shortcuts;
  }
```

Reply:
OK, I see. Can we mark the `manager` getter as non-nullable and return `widget.manager ?? _internalManager!` from it?  The `manager` getter shouldn't ever return null when accessed, since one or the other of `_internalManager` and `widget.manager` should always be non-null.

Reply:
Done